yapapi/rest/configuration.py

```diff
@@ -48,33 +65,40 @@ def resolve_url(given_url: Optional[str], env_val: str, prefix: str) -> str:
 
     @property
     def app_key(self) -> str:
+        """yagna daemon's application key used to access the REST API"""
         return self.__app_key
 
     @property
     def market_url(self) -> str:
+        """The URL of the Market REST API"""
```

Comment:
I hate putting full stops at the end of docstrings that are not sentences (like this one). But have them in other places so maybe we should put one here as well?

Reply:
I wouldn't do that